core/src/main/resources/error/error-classes.json

```diff
@@ -1266,6 +1266,11 @@
           "DISTRIBUTE BY clause."
         ]
       },
+      "GENERATED_COLUMN_UNSUPPORTED_FOR_PROVIDER" : {
+        "message" : [
+          "Target data source with table provider: \"<provider>\" does not support creating generated columns with GENERATED ALWAYS AS expressions in <statement>."
```

Comment:
Can we reuse the `TABLE_OPERATION` error class? The final error message can be `Table xxx does not support generated columns`.